lib/browser/api/extensions.js

```diff
@@ -30,6 +30,7 @@ process.on('EXTENSION_READY_INTERNAL', (installInfo) => {
     }
     browserActions.setDefaultBrowserAction(installInfo.id, details)
     addBackgroundPageEvent(installInfo.id, 'chrome-browser-action-clicked')
+    addBackgroundPageEvent(installInfo.id, 'chrome-context-menus-clicked')
```

Comment:
this will only add the context menu events if the extension also has a browser action. Just moving it out of the `if` should be fine